src/MassTransit/DiagnosticSourceExtensions.cs

```diff
@@ -24,6 +24,9 @@ public static void AddSendContextHeaders(this StartedActivity? startedActivity,
             activity.AddTag(DiagnosticHeaders.InitiatorId, context.InitiatorId);
             activity.AddTag(DiagnosticHeaders.SourceAddress, context.SourceAddress);
             activity.AddTag(DiagnosticHeaders.DestinationAddress, context.DestinationAddress);
+            activity.AddTag("span.kind", "producer");
```

Comment:
could you please move it to DiagnosticHeaders?
also we have our own so we could change them to use right convention, for eg: `DiagnosticHeaders.DestinationAddress`.

Reply:
Updated - i've not updated existing headers though. Unsure what's currently using them and unsure the naming doesn't completely align. What you think on the naming i've gave?

Reply:
"span.kind" -> DiagnosticHeaders.ServiceKind
DiagnosticHeaders.DestinationAddress -> "peer.address"

Reply:
Updated